bats/tests/helpers/load.bash

```diff
@@ -10,6 +10,12 @@ PATH_BATS_ROOT=$(
     pwd
 )
 
+# Use fatal() to abort loading helpers; don't run any tests
```

Comment:
`bats-support` has a `fail`; would that be useful?

Reply:
I don't think so, as I called my version `fail` initially and then spend some time to figure out why it wouldn't work. If the bat-support version had worked, then I probably wouldn't have noticed. 😄 But I can take another look...

Reply:
I checked again, and the problem is that the bats `fail` function writes to stderr, which is discarded unless a test failure is reported; we need to write to `&3` to print something that is not captured again by bats.

Reply:
Ah, I thought `fail` would _also_ cause the test to fail. If it didn't do that, that's… not useful.